community-operators/vault/vaultoperator.v0.4.10.clusterserviceversion.yaml

```diff
@@ -0,0 +1,269 @@
+#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
+#! parse-kind: ClusterServiceVersion
+apiVersion: vault.banzaicloud.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: vaultoperator.v0.4.10
+  namespace: placeholder
+  annotations:
```

Comment:
Additional annotations for CSVs can be found here: https://github.com/operator-framework/community-operators/blob/master/docs/required-fields.md
These fields help to improve the Operator Hub UI in OpenShift's console.

Reply:
I have filled all the required annotations according to the documentation now, thanks!